Podcast/Episode.swift

```diff
@@ -14,6 +14,7 @@ class Episode: NSObject {
     var title: String = ""
     var series: Series?
     var dateCreated: Date?
+    var descriptionText: String?
```

Comment:
If you add in the initializer with more arguments, attempt to make these non-optional.